### PR TITLE
Minor remote viewpoint bugfixes.

### PR DESCRIPTION
**Pull request by SteelTitanium**
Fixes https://mb.srb2.org/showthread.php?p=802600#post802600 and by extension https://mb.srb2.org/showthread.php?t=42918. Where post-processing visual effects won't work correctly from a remote viewpoint. 

I was given permission by Jimita to do this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,5 @@ Win32_LIB_ASM_Release
 *.user
 *.db
 *.opendb
+*.bat
```
Comment (alama) [inline .gitignore]: Why is this here?

Comment (monster-psychic-cat) [reply .gitignore]: No it's not
